docs/tasks/administer-cluster/ip-masq-agent.md

```diff
@@ -0,0 +1,97 @@
+---
+title: IP Masquerade Agent User Guide
+---
+
+{% capture overview %}
+This page show how to configure and enable the ip-masq-agent. 
+{% endcapture %}
+
+{% capture prerequisites %}
+Kubernetes 1.7
```

Comment:
Probably should make this:
<pre>
* Kubernetes 1.7
* {% include task-tutorial-prereqs.md %}
</pre>